src/edge/views.py

```diff
@@ -10,20 +11,39 @@
 
 from edge.models import Fragment, Genome, Operation
 from edge.io import IO
+from edge import import_gff
+from edge.tasks import build_genome_blastdb
 
 
 def genome_export(request, genome_id):
     get_genome_or_404(genome_id)
     io = IO(Genome.objects.get(pk=genome_id))
-
     response = HttpResponse(content_type='text/plain')
     response['Content-Disposition'] = "attachment; filename=\"g%s.gff\"" % genome_id
     io.to_gff_file(response)
     return response
 
 
+def genome_import(request):
+    import tempfile
+    res = {
+      "imported_genomes": [],
+    }
+    for name in request.FILES:
+        with tempfile.NamedTemporaryFile(mode='rw+b', delete=False) as gff:
+            gff.write(request.FILES.get(name).read())
+            gff.close()
+        g = import_gff(name, gff.name)
+        os.unlink(gff.name)
+        res["imported_genomes"].append({
+            "id": g.id,
+            "name": g.name,
+        })
+        build_genome_blastdb.delay(g.id)
```

Comment:
you should return the task id in the API, so that if someone wants to check on the status, they can